app/actors/hyrax/actors/apply_order_actor.rb

```diff
@@ -16,44 +17,28 @@ def can_edit_both_works?(env, work)
           env.current_ability.can?(:edit, work) && env.current_ability.can?(:edit, env.curation_concern)
         end
 
-        def sync_members(env, ordered_member_ids)
-          return true if ordered_member_ids.nil?
-          cleanup_ids_to_remove_from_curation_concern(env.curation_concern, ordered_member_ids)
-          add_new_work_ids_not_already_in_curation_concern(env, ordered_member_ids)
-          env.curation_concern.errors[:ordered_member_ids].empty?
+        def sync_members(env, member_ids)
+          return true if member_ids.nil?
+          env.change_set.validate(member_ids: member_ids.select { |work_id| can_add_work_id?(env, work_id) })
+          env.change_set.sync
+          persister.save(resource: env.curation_concern)
```

Comment:
I'm not sure that it's clear that `sync_members` is going to cause a save operation.  Is it possible to do all the saving in the BaseActor?  That's what we use to do. Do we still need both a BaseActor and ApplyOrderActor?

Reply:
I don't even know if ApplyOrderActor is required any more, since order is handled farther down by Valkyrie. Whatever order you set the members in, that's the order they persist in.

Reply:
I think we should yank it out then.